Makefile

```diff
@@ -38,7 +38,7 @@ check_license:
 
 test:
 	@echo ">> running tests"
-	@$(GO) test -short $(pkgs)
+	@$(GO) test -race -short $(pkgs)
```

Comment:
What do you suggest in this case then? If we add the race detector here, the CI tests will run for ages on CircleCI.

Just revert the -race then?